src/Methods/Address/GetSpendables.ts

```diff
@@ -9,29 +12,47 @@ import { btcToSat } from "../../Utilities/Bitcoin";
 
 const MAX_SPENDABLES = 200;
 
+async function outputHasRunes(outpoint: string): Promise<boolean> {
+  const [txid, n] = outpoint.split(":");
+  const tx = await getTransaction(txid);
+  if (!tx) return false;
+  const decipher = tryDecodeRunestone(tx);
+  if (!decipher) return false;
+  if (!isRunestone(decipher)) return false;
+
+  const pointer = decipher.pointer ?? 1;
+  if (decipher.etching || decipher.mint) {
+    if (Number(n) === pointer) return true;
+  }
+  if (decipher.edicts) {
+    decipher.edicts.forEach((edict) => {
+      if (Number(n) === edict.output) return true;
+    });
+  }
+
+  return false;
+}
+
 export default method({
   params: Schema({
     address: string,
     value: number,
     safetospend: boolean.optional(),
     filter: array.of(string).optional(),
+    limit: number.optional(),
   }),
-  handler: async ({ address, value, safetospend = true, filter = [] }) => {
+  handler: async ({ address, value, safetospend = true, filter = [], limit }) => {
     const spendables = [];
 
     let totalValue = 0;
     let safeToSpend = 0;
     let scanned = 0;
 
-    const cursor = db.outputs.collection.find({ addresses: address, ...noSpentsFilter }, { sort: { value: -1 } });
-    while (await cursor.hasNext()) {
-      const output = await cursor.next();
-      if (output === null) {
-        continue;
-      }
-
+    const outputs = await db.outputs.collection
+      .find({ addresses: address, ...noSpentsFilter }, { sort: { value: -1 } })
+      .toArray();
```

Comment:
I concur with the issue @kevzzsk mentioned on Slack, doing `toArray()` here would take up a lot of memory, especially during moments of high traffic. We can do a `.batchSize(10)` and retain the cursor, to reduce the memory footprint.